src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationConsentedMidHandler.java

```diff
@@ -29,15 +42,21 @@ public boolean canHandle(CallbackType callbackType, CaseType caseType, EventType
     }
 
     @Override
-    public GenericAboutToStartOrSubmitCallbackResponse<Map<String, Object>> handle(CallbackRequest callbackRequest,
+    public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(FinremCallbackRequest callbackRequest,
```

Comment:
There's a simple issue of neatness here. Compare how you've written this class compare to how you've written UpdateContactDetailsConsentedMidHandler. Couldn't they be consistent?
So for example line 49
`List<String> validate = postalService.validate(callbackRequest.getCaseDetails().getData());`

then later on line 53
`FinremCaseData caseData = finremCaseDetails.getData();`